src/runtime/converterextensions.cs

```diff
@@ -156,7 +156,7 @@ bool TryDecode(IntPtr pyHandle, out object result)
 
         #endregion
 
-        internal static void Reset()
+        public static void Reset()
```

Comment:
Is there a scenario when this should be done outside of runtime shutdown?

Reply:
@lostmsu Doubt it.  I just needed it to clear the state between unit test runs.

Reply:
Yes, I think you should be shutting down the runtime.